app/controllers/slack_authorizations_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class SlackAuthorizationsController < ApplicationController
```

Comment:
please move to separate namespace - `Slack::AuthorizationsController`

Reply:
See no reasons for it. But ok, will do it